nupic/algorithms/CLAClassifier.py

```diff
@@ -213,6 +213,31 @@ def __setstate__(self, state):
     self._version = BitHistory.__VERSION__
 
 
+  def write(self, proto):
+    proto.id = self._id
+
+    statsProto = proto.init("stats", len(self._stats))
```

Comment:
It isn't necessarily contiguous so probably should be a map. Actually for RDSE the bucket indices can be positive or negative I believe so it would have to be a map.

https://github.com/numenta/nupic/issues/2106